cmd/util/cmd/compare-cadence-vm/cmd.go

```diff
@@ -63,7 +65,7 @@ func init() {
 
 	Cmd.Flags().IntVar(&flagBlockCount, "block-count", 1, "number of blocks to process (default: 1)")
 
-	Cmd.Flags().BoolVar(&flagPrintTraces, "print-traces", false, "print traces for mismatched transactions")
+	Cmd.Flags().BoolVar(&flagWriteTraces, "write-traces", false, "write traces for mismatched transactions")
```

Comment:
I am approving this PR, but wanted to emphasize that I do _not_ know whether change of command line parameter name breaks anything. If you know that it doesn't break anything, please feel free to disregard this comment. But if you are unsure, please make sure you ask somebody who does know.